Marlin/src/pins/stm32f1/pins_ZM3E4_V2_0.h

```diff
@@ -0,0 +1,317 @@
+/**
+ * Marlin 3D Printer Firmware
+ * Copyright (c) 2021 MarlinFirmware [https://github.com/MarlinFirmware/Marlin]
+ *
+ * Based on Sprinter and grbl.
+ * Copyright (c) 2011 Camiel Gubbels / Erik van der Zalm
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+#pragma once
+
+#include "env_validate.h"
+
+#define BOARD_INFO_NAME "ZONESTAR ZM3E4 V2.0"
+
+// =====================NOTE=====================
+// #define TONE_TIMER     1
+// #define TONE_CHANNEL   2
+// Please redefine the TONE_TIMER and TONE_CHANNEL in
+// .platformio\packages\framework-arduinoststm32-maple\STM32F1\cores\maple\tone.cpp
+// line 31 and line      34
+// =====================NOTE=====================
```

Comment:
you can define those in env build_flags -DTONE_TIMER=1 -DTONE_CHANNEL=2

Reply:
Updated